entity/entity_pass.cc

```diff
@@ -103,22 +108,37 @@ EntityPass* EntityPass::AddSubpass(std::unique_ptr<EntityPass> pass) {
 }
 
 bool EntityPass::Render(ContentContext& renderer,
-                        RenderPass& parent_pass) const {
+                        RenderPass& parent_pass,
+                        Point position) const {
   TRACE_EVENT0("impeller", "EntityPass::Render");
 
-  for (const auto& entity : entities_) {
-    if (!entity.Render(renderer, parent_pass)) {
-      return false;
+  if (position.IsZero()) {
```

Comment:
This seems like an unnecessary optimization to prevent a copy of entities (which we are assuming are cheap to copy). TBH, I'd get rid of it for code clarity.

Reply:
Done.